packages/libs/wallet-adapter-core/src/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from "./types";
+export * from "./BaseWalletAdapter";
+export * from "./WalletAdapterClient";
+export * from "./utils/sign";
+export * from "./utils/type-helpers";
+export { StandardSchemaV1 } from "@standard-schema/spec";
```

Comment:
Why is this exported? Shouldn't this be available through the library for others to use?